Sources/MapboxNavigation/ArrivalController.swift

```diff
@@ -0,0 +1,171 @@
+
+import UIKit
+import MapboxDirections
+import MapboxCoreNavigation
+import MapboxMobileEvents
+
+
+class ArrivalController {
```

Comment:
I've documented  new components. Right now this class is just about `EndOfRouteViewController`. If in future it changes, I think it would be easy to update `ArrivalController` too.